### PR TITLE
Created stats component

### LINKED ISSUE #24: [Frontend] Create the stats component
# Description

### Refer the ui design and create the stats component

# Screenshot

image
image

# Colours and Information
image
# link

For reference you can see the design prototype : 
https://www.figma.com/proto/dMw9GAeJja2baVhDoceAEB/Untitled?node-id=9%3A48&viewport=-1544%2C148%2C0.4358828365802765&scaling=contain

### PR DESCRIPTION
**Pull request by AdityaTeltia**
### Description

Designed and created stats component inside ```src/components``` with the name ```Stats.js```.

### Related Issue

Fixes #24 

### Type of Change:

- Code
- User Interface
- New Feature

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?

Tested by rendering the component in local machine.

### Checklist:

- [x] Pull request is made against `development` branch.
- [x] My code follows the style guidelines of this project.
- [x] I run `npm run format` before creating the pull request.
- [x] I have performed a self-review of my own code.

### Screenshots (if appropriate):
stats

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
@AdityaTeltia Can you provide the full page screenshot

**Comment by AdityaTeltia** [comment]
> @AdityaTeltia Can you provide the full page screenshot

localhost_3000_ (2)

**Comment by AdityaTeltia** [comment]
We just have to add this component in ```Navigation.js``` component that I merged earlier in #21 and We will get this , Also I don't think issue #22 needs to be there anymore as I have pushed the` header` component along with `Navigation` one.
@Abhishek-kumar09

**Comment by Abhishek-kumar09** [comment]
> We just have to add this component in Navigation.js

Have you done it? If not please do.

**Comment by AdityaTeltia** [comment]
> > We just have to add this component in Navigation.js
> 
> Have you done it? If not please do.

Actually , In my `temp2` branch the `navigation` component is not present as I was working parallely on two issues and the `navigation` component got merged.
So should I create a new PR with a new branch or there are any commands so that I can even my `temp2` branch with `origin`

**Comment by Abhishek-kumar09** [comment]
you can do:
```
git merge navigation
```
It may help.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -18,9 +18,7 @@ A clear description of what the actual behavior is.
 
 ## Steps to Reproduce the Problem
 
-1.
-2.
-3.
+1. 2. 3.
```
Comment (Abhishek-kumar09) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please revert this.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Pull request merged**